printer_mods/arhodiewithsoul/Gantry/X_Axis/X_Carriage/readme.md

```diff
@@ -0,0 +1,8 @@
+Contains updated X Carriage mounts (and connector cover) to try assist in packaging connectors under the connector cover.
+
+![x_carriage_frame_right_V1 8_001 (Large)](https://user-images.githubusercontent.com/80538348/116250629-ca26ad80-a765-11eb-9a9f-608dbce2e904.jpg)
+![x_carriage_frame_right_V1 8_002 (Large)](https://user-images.githubusercontent.com/80538348/116250636-cb57da80-a765-11eb-81fd-3641de3dad2c.jpg)
+
+![a _connector_cover_V1 8_001 (Large)](https://user-images.githubusercontent.com/80538348/116250665-d4e14280-a765-11eb-9a49-3b280671d830.jpg)
+![a _connector_cover_V1 8_002 (Large)](https://user-images.githubusercontent.com/80538348/116250674-d6ab0600-a765-11eb-8fea-332934105ef9.jpg)
+![a _connector_cover_V1 8_003 (Large)](https://user-images.githubusercontent.com/80538348/116250681-d7dc3300-a765-11eb-9db1-3247a388856e.jpg)
```

Comment:
`[a]_connector_cover_V1.8.stl` is not in a printable orientation.

Reply:
Updated